app/Jobs/CheckTraefikVersionForServerJob.php

```
@@ -0,0 +1,156 @@
+        if (str_contains(strtolower(trim($imageTag)), ':latest')) {
+            return;
+        }
+
+        // Parse current version to extract major.minor.patch
+        $current = ltrim($currentVersion, 'v');
+        if (! preg_match('/^(\d+\.\d+)\.(\d+)$/', $current, $matches)) {
+            return;
+        }
+
+        $currentBranch = $matches[1]; // e.g., "3.6"
+        $currentPatch = $matches[2];  // e.g., "0"
```

PHPMD (2.15.0)

Avoid unused local variables such as '$currentPatch'. (undefined)

(UnusedLocalVariable)